build.gradle

```diff
@@ -33,7 +34,7 @@ subprojects {
     version = "0.0.1"
 
     ext {
-        mesosVer = "0.21.1"
+        mesosVer = "0.22.1"
         hadoopVer = "2.7.0"
```

Comment:
Probably not a good idea to do implicit version changes without a feature that requires it.